storage/backend/ceph/ceph_test.go

```diff
@@ -109,6 +128,20 @@ func (s *cephSuite) TestMountUnmountVolume(c *C) {
 	s.readWriteTest(c, mp)
 	c.Assert(mountDriver.Unmount(driverOpts), IsNil)
 	c.Assert(crudDriver.Destroy(driverOpts), IsNil)
+
+	driverOpts = storage.DriverOptions{
+		Volume:    volumeSpecTestPool,
+		FSOptions: filesystems["ext4"],
+		Timeout:   5 * time.Second,
+	}
+
+	if done {
+		return
+	}
+
+	done = true
+
+	goto again
```

Comment:
In case of successful completion during the first run, why are we preforming another loop of execution rather returning? 

What is the actual intention of using goto here?